src/binding/fortran/use_mpi/buildiface

```diff
@@ -696,9 +696,9 @@ $PRIVATEVAR = "";
 print MPIFD <<EOT;
         TYPE$BINDACCESS :: MPI_Status
            $BINDDEF
-           INTEGER$PUBLICVAR :: MPI_SOURCE, MPI_TAG, MPI_ERROR
            INTEGER$PRIVATEVAR :: count_lo
            INTEGER$PRIVATEVAR :: count_hi_and_cancelled
+           INTEGER$PUBLICVAR :: MPI_SOURCE, MPI_TAG, MPI_ERROR
```

Comment:
reference in `mpi.h.in`: https://github.com/pmodels/mpich/blob/96ed11354ac158ea31a4d8b1d5d567c0fdb3fb0b/src/include/mpi.h.in#L578-L586